cmd/fyne/internal/commands/build.go

```diff
@@ -90,6 +90,13 @@ func Build() *cli.Command {
 // Build parse the tags and start building
 func (b *Builder) Build() error {
 	if b.srcdir != "" {
+		// Use absolute srcdir
```

Comment:
It would be better to check and correct `srcdir` earlier so that it is consistent (validate etc can also use this).
Build will not always be called so later stages will not have this correction run.
It may also avoid the duplication in the PR.

Reply:
Builder.Build function is the first thing called when user starts fyne build and only called by this command. I don't see anywhere earlier than this. Other commands just calls underlying build function. Thats why I have to add it to Package.validate because it seems release and install commands also calling it and we don't need to add another correction for them.

Reply:
Yes, we do have `fyne build` command that try to mimick the `go build` command.

Reply:
> Builder.Build function is the first thing called when user starts fyne build and only called by this command. I don't see anywhere earlier than this. Other commands just calls underlying build function

What I meant is that this may not be called. If the exe specified already exists then it won't try to build.